internal/cmdget/cmdget_test.go

```diff
@@ -355,8 +356,8 @@ func TestCmd_Execute_flagAndArgParsing(t *testing.T) {
 			runE: NoOpRunE,
 			validations: func(repo, dir string, r *cmdget.Runner, err error) {
 				assert.NoError(t, err)
-				assert.Equal(t, fmt.Sprintf("file://%s", repo), r.Get.Git.Repo)
-				assert.Equal(t, "master", r.Get.Git.Ref)
+				assert.Equal(t, fmt.Sprintf("file://%s", repo), r.Get.Upstream.(location.Git).Repo)
+				assert.Equal(t, "master", r.Get.Upstream.(location.Git).Ref)
```

Comment:
So far, I am only seeing `git` related tests here. Do we plan to add `oci` specific tests here ?